pyeudiw/tests/satosa/test_backend.py

```diff
@@ -21,6 +21,8 @@
                             load_specification_from_yaml_string)
 from pyeudiw.tools.utils import exp_from_now, iat_now
 
+from pyeudiw.storage.exceptions import ReplicaError
```

Comment:
this is not a Replica error since it doesn't take in account any replication mechanisms but just the missing trust attestation, please change accordingly this exception

Reply:
Resolved in [302d523](https://github.com/italia/eudi-wallet-it-python/pull/94/commits/302d523f5d36cafab1c3c04c049a27d06c21d575)